github/orgs_teams.go

```diff
@@ -377,3 +393,28 @@ func (s *OrganizationsService) RemoveTeamMembership(team int, user string) (*Res
 
 	return s.client.Do(req, nil)
 }
+
+// ListPendingTeamInvitations get pending invitaion list in team.
+// Warning: The API may change without advance notice during the preview period.
+// Preview features are not supported for production use.
+//
+// GitHub API docs: https://developer.github.com/v3/orgs/teams/#list-pending-team-invitations
+func (s *OrganizationsService) ListPendingTeamInvitations(team int, opt *ListOptions) ([]*Invitation, *Response, error) {
+	u := fmt.Sprintf("teams/%v/invitations", team)
+	u, err := addOptions(u, opt)
+
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req.Header.Set("Accept", mediaTypeOrgMembershipPreview)
```

Comment:
Please add a `// TODO: remove custom Accept header when this API fully launches.` comment here, just like all other instances.